tools/spin-ec2/main.go

```diff
@@ -112,36 +89,38 @@ func interactiveMode() {
 			continue
 		}
 
-		break
-	}
+		for _, cOpt := range chosenProvisionOptions{
+			if cOpt.licenseKeyRequired {
+				isLicenseRequired = true
+			}
+		}
 
-	u, err := user.Current()
-	if err != nil {
-		log.Fatalf(err.Error())
+		break
 	}
-	provisionHostPrefix = fmt.Sprintf("%s-%s", u.Username, provisionHostPrefix)
 
-	fmt.Printf("Chosen AMIs\n\n")
-	for _, chosenOption := range chosenOptions {
-		printVmInfo(chosenOption, provisionHostPrefix, chosenProvisionOptions)
-	}
-	confirm := askUser(fmt.Sprintf("Is this right [(%s)es / (%s)o / (%s)uit]: ", colorizeGreen("y"), colorizeYellow("n"), colorizeRed("q")))
+	license := ""
 
-	if !(confirm == "" || confirm == "yes" || confirm == "y") {
-		exit()
+	if isLicenseRequired {
+		license = askUser("NR license key required for chosen provision option(s): ")
 	}
 
-	prepareAnsibleConfig(chosenOptions, provisionHostPrefix)
-
 	curPath, err := os.Getwd()
 	if err != nil {
 		panic(err)
 	}
 
-	execNameArgs("ansible-playbook",
-		"-i", path.Join(curPath, "test/automated/ansible/inventory.local"),
-		"--extra-vars", "@"+path.Join(curPath, inventory),
-		path.Join(curPath, "test/automated/ansible/provision.yml"))
+	if !skipVMCreation{
+		execNameArgs("ansible-playbook",
+			"-i", path.Join(curPath, inventoryLocal),
+			"--extra-vars", "@"+path.Join(curPath, inventoryForCreation),
+			path.Join(curPath, "test/automated/ansible/provision.yml"))
+
+		execNameArgs("ansible-playbook",
+			"-i", path.Join(curPath, inventoryLinux),
+			path.Join(curPath, "/test/automated/ansible/install-requirements.yml"))
```

Comment:
maybe is not giving an error but I would remove the leading slash to be consistent with line 116
`path.Join(curPath, "test/automated/ansible/install-requirements.yml"))`

Reply:
done